python/src/aac/plugins/plugin_manager.py

```diff
@@ -1,9 +1,9 @@
 """Provide access to plugins and plugin data."""
```

Comment:
This is fantastic! This helps resolve the issue of having to "restart" aac to pick up when new plugins have been added it seems. So cool!